CODEOWNERS

```diff
@@ -77,6 +77,8 @@ Dockerfile* @cilium/build
 /hubble-relay.Dockerfile @cilium/hubble
 /images @cilium/build
 /install/kubernetes/ @cilium/kubernetes @cilium/docs @cilium/helm
+/install/kubernetes/cilium/charts/hubble-relay @cilium/hubble
+/install/kubernetes/cilium/charts/hubble-ui @cilium/hubble
```

Comment:
Should @cilium/helm also be a codeowner on these files?

Reply:
@cilium/helm is already a codeowner of `/install/kubernetes/`, see line above.

Reply:
Oh wait, are you saying that the later match takes precedence so this group would be excluded? If yes then I'll add @cilium/helm

Reply:
Yes, the later match takes precedence and overwrites any previous match.

Reply:
Maybe we also need to add @cilium/kubernetes, though it's starting to be quite a party...

Reply:
Updated so that it matches the groups from parent + @cilium/hubble.